examples/micro_speech/main/audio_provider.cc

```diff
@@ -148,7 +148,7 @@ TfLiteStatus InitAudioRecording() {
   }
   /* create CaptureSamples Task which will get the i2s_data from mic and fill it
    * in the ring buffer */
-  xTaskCreate(CaptureSamples, "CaptureSamples", 1024 * 32, NULL, 10, NULL);
```

Comment:
The original stack size was 32KB, this is rather big, this potentially starve other task of memory when porting over to other code base, this happened for me.